### PR TITLE
Update README (add Crowdin)

### PR DESCRIPTION
**Pull request by ovari**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,9 +29,12 @@ unzip phoenicis-dist.zip
 For more details (e.g. regarding dependencies) consider the [documentation](https://phoenicisorg.github.io/phoenicis/).
 
 ## Scripts
-Please make your pull request on this repository: https://github.com/PhoenicisOrg/scripts to add scripts
+To add or update scripts, please make pull requests to this repository: https://github.com/PhoenicisOrg/scripts
+
+## Translate
+Phoenicis is localized using crowdin: https://crowdin.com/project/phoenicis
```
Comment (plata) [inline README.md]: Shouldn't it be "Crowdin"?

Comment (ovari) [reply README.md]: Thanks. Should be fixed

**Comment by ovari** [comment]
@plata do you know when Travis CI will build successfully?

Can we delete our fork?

Thank you

**Comment by plata** [comment]
Please keep the fork until the PR is merged. I'm investigating the Travis issue currently.

**Pull request merged**